packages/fxa-auth-server/config/dev.json

```diff
@@ -206,7 +206,9 @@
     },
     "clientIdToServiceNames": {
       "dcdb5ae7add825d2": "123done",
-      "98e6508e88680e1a": "fxa-settings"
+      "98e6508e88680e1a": "fxa-settings",
+      "7377719276ad44ee": "pocket-mobile",
+      "749818d3f2e7857f": "pocket-web"
```

Comment:
In order to "quarantine" pocket access tokens in mysql I opted to use this existing bit of config. These are already assigned in prod here: https://github.com/mozilla-services/cloudops-deployment/blob/8adf05f820c05fe6b29b315e557df9afe3baaeba/projects/fxa/puppet/yaml/app/fxa.prod.yaml